src/PKSim.Core/Snapshots/Mappers/SimulationComparisonMapper.cs

```diff
@@ -0,0 +1,109 @@
+      }
+
+      private async Task updateIndividualComparisonSnapshot(SimulationComparison snapshot, ISimulationComparison simulationComparison)
+      {
+         var individualSimulationComparison = simulationComparison as IndividualSimulationComparison;
+         if (individualSimulationComparison == null)
+            return;
+
+         snapshot.IndividualComparison = await _individualSimulationComparisonMapper.MapToSnapshot(individualSimulationComparison);
+      }
+
+      public override async Task<ISimulationComparison> MapToModel(SimulationComparison snapshot, PKSimProject project)
+      {
+         var simulationComparison = await createSimulationComparisonFrom(snapshot, project);
+         MapSnapshotPropertiesToModel(snapshot, simulationComparison);
+         simulationComparison.IsLoaded = true;
+         return simulationComparison;
+      }
+
+      private async Task<ISimulationComparison> createSimulationComparisonFrom(SimulationComparison snapshot, PKSimProject project)
+      {
+         if (snapshot.IndividualComparison != null)
+         {
+            var individualSimulationComparison = await createIndividualComparisonModel(snapshot, project);
+            return addSimulationsToComparison(individualSimulationComparison, snapshot, project);
+         }
+
+         var populationSimulationComparison = _objectBaseFactory.Create<PopulationSimulationComparison>();
+         await updatePopulationComparisonModel(populationSimulationComparison, snapshot, project);
+         return addSimulationsToComparison(populationSimulationComparison, snapshot, project);
+      }
+
+      private async Task updatePopulationComparisonModel(PopulationSimulationComparison populationSimulationComparison, SimulationComparison snapshot, PKSimProject project)
+      {
+         var simulationComparisonContext = new SimulationAnalysisContext(project.AllObservedData);
+         var allPopulationAnalysis = await _populationAnalysisChartMapper.MapToModels(snapshot.PopulationComparisons, simulationComparisonContext);
+         allPopulationAnalysis.Each(populationSimulationComparison.AddAnalysis);
+         populationSimulationComparison.ReferenceGroupingItem = snapshot.ReferenceGroupingItem;
+         populationSimulationComparison.ReferenceSimulation = project.BuildingBlockByName<PopulationSimulation>(snapshot.ReferenceSimulation);
+      }
+
+      private Task<IndividualSimulationComparison> createIndividualComparisonModel(SimulationComparison snapshot, PKSimProject project)
+      {
+         var simulationComparisonContext = new SimulationAnalysisContext(project.AllObservedData);
+         var comparedSimulations = allComparedSimulationsFrom<IndividualSimulation>(snapshot, project);
+         comparedSimulations.Each(s => simulationComparisonContext.AddDataRepository(s.DataRepository));
+         return _individualSimulationComparisonMapper.MapToModel(snapshot.IndividualComparison, simulationComparisonContext);
```

Comment:
The Chart is ACTUALLY the comparison. But we save it as a subnode of the Comparison Snapshot. This is useful to only have one type of Comparison representation in Json